homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -1099,7 +980,8 @@ def create_powerview_shade_entity(
         shade.capability.type, (PowerViewShade,)
     )
     _LOGGER.debug(
```

Comment:
Unrelated to this PR, but it would be nicer to make `TYPE_TO_CLASSES` use an https://docs.python.org/3/library/enum.html#enum.IntEnum